src/main/kotlin/racingcar/controller/RacingController.kt

```diff
@@ -0,0 +1,13 @@
+package racingcar.controller
+
+import racingcar.model.Racing
+import racingcar.utils.Validation
+import racingcar.view.Input
+
+class RacingController {
+    fun playGame() {
+        val carName = Input().getCars()
+        val tryCount = Input().getCount()
+        Racing().race(tryCount, Racing().readyToRace(carName))
```

Comment:
`Input()`과 `Racing()`을 통해서 객체를 계속 생성하는데 계속해서 다른 객체를 사용하는 것으로 보입니다.
객체를 생성하고 해당 객체를 여러번 사용하는 방법은 어떨까요?
```suggestion
class RacingController {
    private val input = Input()
    private val racing= Racing()
    fun playGame() {
        val carName = input.getCars()
        val tryCount = input.getCount()
        val cars = racing.readyToRace(carName)
        racing.race(tryCount, cars)
```

Reply:
그렇네요!! 재사용하고 있지 않았다니 충격입니다..
좋은 리뷰 감사합니다.